src/__tests__/test_utils.ts

```diff
@@ -0,0 +1,40 @@
+import { grader, student } from './examples/util'
+import { awsEventFactory } from './helpers'
+import { runAll } from '../index'
+
+const makeAwsEvent = awsEventFactory({
+  chapter: 1,
+  external: {
+    name: 'NONE',
+    symbols: [
+      '__track_function',
+      '__reset_function_count',
+      '__get_function_count',
+      'make_funny',
+    ]
+  },
+  globals: [
+    ['__tracking_function', 'global.__track_function("make_funny");'],
```

Comment:
This global doesn't seem to be used, is it required?

Reply:
We just need the __track_function to be evaluated during the globals initialization. The actual value isn't used.